test/helper/attribute.ex

```diff
@@ -4,4 +4,8 @@ defmodule Jerboa.Test.Helper.Attribute do
   def total(x) do
     x |> Keyword.values |> Enum.sum
   end
+
+  def value_length(<<_::16, byte_length::16, _::size(byte_length)-bytes>>), do: 8 * byte_length
```

Comment:
1. `check_length`
2. `verify_value_length/2` - pass a binary and expected value to this function, which will return true or false

```elixir
assert verify_value_length(binary, 32)
```